docs/README.md

```diff
@@ -19,8 +18,21 @@ If the files in `pkg/apis` are updated we need to run `codegen` scripts
 
 ## Running Operator (Development)
 
-1. Set `KO_DOCKER_ENV` environment variable ([ko#usage](https://github.com/google/ko#usage))
+### Setup
+- Set `KO_DOCKER_ENV` environment variable ([ko#usage](https://github.com/google/ko#usage))
+
+### Run operator
+
+**Target: Kubernetes**
+```shell script
+    make apply
+```
+
+**Target Openshift**
+```shell script
+    TARGET=openshift make apply
```

Comment:
or `make TARGET=openshift apply`, both work I think :stuck_out_tongue_closed_eyes:

Reply:
:heavy_check_mark: for uniformity